app/views/offer_mailer/offer_accepted_email.html.erb

```diff
@@ -12,5 +12,9 @@
 <p>
   Retain the login id: <%= @user.email %> and password that you created
   during the application process. You will need it to refer back to the
-  application web site to review the events associated with your application.    
-</p>
\ No newline at end of file
+  application web site to review the events associated with your application.
+</p>
+<p>
+  Please review the
+  <%= link_to "MMSS #{@camp_config.year} Student Information Packet", "https://docs.google.com/document/d/1bIzIKld_Lff8HpkAi2WPRj6hfthRW98y50_nXmJtLB8/edit?usp=sharing" %> for more information regarding program policies and procedures.
+</p>
```

Comment:
Why is the Student Information Packet address hardcoded? CampConfiguration model has student_packet_url. Why did we remove it from the Active Admin CampConfiguration page? It doesn't make sense.

Reply:
We removed the student_packet_url field form the Camp Configuration view in AA when we removed teh upload of the student_packet.
I was on the fence on whether or not to use the static URL to the Google doc Kassidy provided or have use the student_packet_url field. I went the other direction with this.
I will update this PR and use the student_packet_url

Reply:
I also fixed the .camp_year in the temaplates.